module/os/linux/zfs/zfs_ctldir.c

```diff
@@ -977,6 +978,15 @@ zfsctl_snapdir_mkdir(struct inode *dip, char *dirname, vattr_t *vap,
 	return (error);
 }
 
+void
```

Comment:
You need to add the `static` keyword kernel to resolve the build warnings.

Reply:
Added just now.